repondeur/zam_repondeur/templates/amendements.html

```diff
@@ -1,152 +1,250 @@
 {% extends "_base.html" %}
 {% import "macros.html" as macros %}
 
+
 {% block header %}
 <style type="text/css">
-    .action i {
-        margin-right: 0.33rem;
+
+    .content {
+        margin: 0 1rem;
+    }
+
+    /* Options avancées */
+    .toggle:before {
+        content: "↓";
+    }
+    .toggle.enabled:before {
+        content: "↑";
+    }
+    #advanced {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        margin: 0 auto 1rem auto;
+        max-width: 56rem;
+    }
+    #advanced > div {
+        flex-basis: 26rem;
+        padding: 2rem;
+        border-bottom: thin solid #979797;
+    }
+    #advanced > div h3 {
+        font-weight: 600;
     }
-    th {
+    #advanced div[role="group"] {
+        margin-top: 1rem;
+        display: flex;
+        flex-wrap: wrap;
+    }
+    #advanced .button {
+        margin: 0 0.75rem 0.75rem 0;
+        flex-basis: 10rem;
+    }
+    #advanced .button:last-child {
+        margin-right: 0;
+    }
+    #advanced .button i {
+        padding-right: 0.5rem;
+    }
+    @media(min-width:1900px) {
+        #advanced {
+            max-width: none;
+        }
+    }
+
+    /* Table formatting */
+    .table th, .table td {
+        vertical-align: middle;
+    }
+    .table thead th {
+        border: none;
+        font-weight: 600;
         white-space: nowrap;
     }
-    th::after {
-        content: " ⇵";
+
+    /* Sticky table header */
+    tr th {
+        position: sticky;
+        position: -webkit-sticky;
+        background: white;
+        z-index: 10;
     }
-    th[data-order="asc"]::after {
-        content: " ↓";
+    tr.headers th {
+        top: 50px;  /* top nav height */
     }
-    th[data-order="desc"]::after {
-        content: " ↑";
+    tr.filters th {
+        top: calc(50px + 48.5px);  /* just below */
     }
-    th.nosort::after {
+
+    /* Sorting */
+    .headers th {
+        cursor: pointer;
+    }
+    .headers th.nosort {
+        cursor: default;
+    }
+    .headers th::before {
+        font-family: Font Awesome\ 5 Free;
```

Comment:
I think that better than escaping the space before the 5 is to double quote the font name?

Reply:
Yes. I think I copied it from a minified version (1 char less!!).